fusion-engine-driver/utils/conversion_utils.hpp

```diff
@@ -220,4 +236,38 @@ class ConversionUtils {
     nmea.sentence = nmea_stream.str();
     return nmea;
   }
+
+  static const std::unordered_map<int, std::string>& getLeapSeconds() {
+    static const std::unordered_map<int, std::string> leap_seconds_ = {
```

Comment:
Make it static constexpr.
constexpr != const:
https://learn.microsoft.com/en-us/cpp/cpp/constexpr-cpp?view=msvc-170